src/components/Comment/index.jsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from "react";
+import comments from "../../data/comments"; 
+import CommentElement from "./CommentElement";
+
+const Comment = ({ postId }) => {
+    const [commentsList, setCommentsList] = useState(comments);
+    const [commentInputValue, setCommentInputValue] = useState("");
+    const [newId, setNewID] = useState(4);
+    const [newComment, setNewComment] = useState({
+        id: 0,
+        content: "",
+        created_at: "",
+        post: postId,
+        author: {
+            id:0,
+            username: "user0",
+        },
+    });
+
+    const handleNewComment = (e) => {
+        setCommentInputValue(e.target.value);
+    };
+    const editOriginalComment = (commentId, editedText) => {
+        const updatedComments = commentsList.map((comment) => {
+            if (comment.id === commentId) {
+                return {...comment, content: editedText};
+            } return comment;
+        });
+        setCommentsList(updatedComments);
+    };
+
+    const handleCommentSubmit = (e) => {
+        e.preventDefault();
+        const createdComment = {
+            ...newComment, 
+            id: newId,
+            content: commentInputValue,
+            created_at: Date.now(),
+        };
+        setNewID(newId + 1);
+        setCommentsList([...commentsList, createdComment]);
+        setCommentInputValue("");
+        alert("댓글 작성"); 
+    };
+
+    const handleCommentDelete = (commentId) => {
+		console.log(commentId);
+        setCommentsList(commentsList.filter((comment) => comment.id !== commentId));
+        alert("댓글 삭제"); 
+    };
+
+    return (
+        <div className="w-full mt-5 self-start">
+            <h1 className="text-3xl font-bold my-5">Comments</h1>
+                // TODO: comments 더미데이터를 돌며 각 댓글마다 CommentElement를 만들어주세요!
+                // Hint: CommentElement에는 댓글인 comment와 댓글 삭제를 하는 handleCommentDelete라는 props가 필요합니다.
```

Comment:
이 부분이 주석이 아니라 텍스트로 들어가 있어서 화면에서 해당 텍스트가 보여요!
리액트에서 return문 내부에서 js 주석을 활용하고 싶으면 {/* ... */}로 감싸주면 됩니다
```suggestion
            {/* TODO: comments 더미데이터를 돌며 각 댓글마다 CommentElement를 만들어주세요! */}
            {/* Hint: CommentElement에는 댓글인 comment와 댓글 삭제를 하는 handleCommentDelete라는 props가 필요합니다. */}
```